.circleci/config.yml

```diff
@@ -59,11 +59,19 @@ jobs:
               git clone git@github.com:BoltApp/integration-tests.git
               cd integration-tests
               npm install
-              mkdir -p /tmp/screenshots
-              SCREENSHOT_DIR=/tmp/screenshots TEST_ENV=sandbox npm run test-checkout-magento1
+              mkdir -p ./screenshots
```

Comment:
Why move the `screenshots` directory? Not that I object, but if we're changing our conventions there should probably be a reason.

Reply:
why is it in tmp? ci creates own space for each sub-task there is no need for tmp.